global-script.lua

```diff
@@ -2030,355 +2030,188 @@ scaleFactors = {
     [true]={name = "Large", position = 1.09, size = 1.1},
     [false]={name = "Standard", position = 1, size = 1},
 }
-posMap = {
+boardLayouts = {
     { -- 1 Board
         ["Balanced"] = {
-            Vector(5.96, 1.08, 16.59),
+            { pos = Vector(5.96, 1.08, 16.59), rot = Vector(0.00, 180.00, 0.00) },
         },
         ["Thematic"] = {
-            Vector(-1.93, 1.08, 20.44), -- NE
+            { pos = Vector(-1.93, 1.08, 20.44), rot = Vector(0.00, 180.00, 0.00), board = "NE" },
         },
     },
     { -- 2 Board
```

Comment:
looks like you're reordering the maps here, can you leave them in this order and naming

Reply:
I think given the community challenge's use of `Fragment 2` we should go with that name, even if the other is better.

Reply:
checked with MJ, and they also prefer `Inverted Fragment` over `Fragment 2`. plus in the near future i'll probably be releasing the community challenge JSON myself so I can always do these conversions. For example we also call the basic layout `Balanced` as opposed to `Standard` which is what reddit CC calls it